dpnp/backend/examples/example_bs_sycl.cpp

```diff
@@ -0,0 +1,141 @@
+#include <CL/sycl.hpp>
+#include <iostream>
+#include <chrono>
+
+cl::sycl::event divide_sycl(cl::sycl::queue q,
+                            const size_t size,
+                            double* price,
+                            double* strike,
+                            double* p_div_s)
+{
+    double* P = price;
+    double* S = strike;
+
+    cl::sycl::event event = q.submit([&](sycl::handler &cgh) {
+	    cgh.parallel_for(size, [=](sycl::item<1> i) { p_div_s[i] = P[i] / S[i];});
+    });
+
+    return event;  
+}
+
+cl::sycl::event log_sycl(cl::sycl::queue q,
+                         cl::sycl::event e,
+                         const size_t size,
+                         double* p_div_s,
+                         double* a)
+{
+    cl::sycl::event event = q.submit([&](sycl::handler &cgh) {
+        cgh.depends_on(e);
```

Comment:
Let's make the code more generic and work with list of event instead of a single event.
`void depends_on(const std::vector<event> &depEvents)`